### DISCUSSION TITLE
Just a question

### DISCUSSION THREAD

**Discussion by Moamin-AbuEwaida** [lauraramirez220612/TeamFourHYF#23] [General]
Do we need to keep the practiced .md files with our names in the repo?? if yes! do we need to keep them in the main repo or could be in another branch??

**Comment by seninet** [comment]
@Moamin-AbuEwaida ,I think that is just for practice only, I don't think we need to keep them

**Reply by Moamin-AbuEwaida** [reply]
yeah, in general i'll close the PR i issued yesterday and everyone can do the same for themselves
